app/azure/Dashboard/DashboardView.tsx

```diff
@@ -0,0 +1,70 @@
+import { useEnvironmentId } from '@/portainer/hooks/useEnvironmentId';
+import { PageHeader } from '@/portainer/components/PageHeader';
+import { DashboardItem } from '@/portainer/components/DashboardItem';
+import { error as notifyError } from '@/portainer/services/notifications';
+import PortainerError from '@/portainer/error';
+import { r2a } from '@/react-tools/react2angular';
+
+import { useResourceGroups, useSubscriptions } from '../queries';
+
+export function DashboardView() {
+  const environmentId = useEnvironmentId();
+
+  const subscriptionsQuery = useSubscriptions(environmentId);
+  if (subscriptionsQuery.isError) {
+    notifyError(
+      'Failure',
+      subscriptionsQuery.error as PortainerError,
+      'Unable to retrieve subscriptions'
+    );
+  }
+
+  const resourceGroupsQuery = useResourceGroups(
+    environmentId,
+    subscriptionsQuery.data
+  );
+  if (resourceGroupsQuery.isError && resourceGroupsQuery.queryErrors) {
+    resourceGroupsQuery.queryErrors.forEach((e) =>
```

Comment:
that can be a lot. why not just notify once if there are any errors? e.g for the first error

Reply:
you're right, no use spamming the UI